templates/components/plugin.js

```diff
@@ -6,7 +6,13 @@ import { wrapFunctional } from './utils'
 const components = {
 <%= components.map(c => {
   const exp = c.export === 'default' ? `c.default || c` : `c['${c.export}']`
-  return `  ${c.pascalName.replace(/^Lazy/, '')}: () => import('../${relativeToBuild(c.filePath)}' /* webpackChunkName: "${c.chunkName}" */).then(c => wrapFunctional(${exp}))`
+  const magicComments = [
+    `webpackChunkName: "${c.chunkName}"`,
+    c.prefetch ? `webpackPrefetch: ${c.prefetch ? 'true' : 'false'}` : false,
```

Comment:
`webpackPrefetch` and `webpackPreload` expected true or a number, so I think we can remove `false`.

Reply:
Thank you for pointing out!
I made the parts more type-strict.